src/stores/session.ts

```diff
@@ -19,27 +14,9 @@ type SessionState = StateTree & {
   googleMapsApiKey: string | null;
 }
 
-type SessionGetters = _GettersTree<SessionState> & {
+const storeName = Store.Session;
 
-  /**
-   * Gets the current theme id.
-   */
-  getTheme(state: SessionState): string;
-}
-
-type SessionActions ={
-
-  /**
-   * Removes a notification.
-   */
-  setTheme(id: string): void;
-}
-
-export type SessionStore = Store<string, SessionState, SessionGetters, SessionActions>;
-
-const storeName = GlobalStore.SESSION;
-
-export default defineStore<typeof storeName, SessionState, SessionGetters, SessionActions>(storeName, {
+export default defineStore(storeName, {
   state: (): SessionState => {
```

Comment:
suggestion: I think this return type is no longer required, with the update TS documentation. (see also `breadcrumb`)